group/templates/group/editGroup.html

```diff
@@ -1,69 +1,43 @@
 {% extends "index/base.html" %}
 {% load static %}
-{% load bootstrap %}
 {% block title_name %}
-
+{% if group %}
+<title>Edit for {{ group.gname }}...</title>
+{% else %}
+<title>New group...</title>
+{% endif %}
 {% endblock title_name %}
- {% block body_block %}
+{% load bootstrap %}
+{% block import_source %}
+<script src="{% static 'multiselect/js/jquery.multi-select.js' %}" type="text/javascript"></script>
+<script src="{% static 'quicksearch/jquery.quicksearch.js' %}" type="text/javascript"></script>
+<script src="{% static 'group/js/editGroup.js' %}" type="text/javascript"></script>
+<link href="{% static 'multiselect/css/multi-select.css'%}" media="screen" rel="stylesheet" type="text/css">
+{% endblock import_source %} 
+{% block body_block %}
 <br>
 <br>
-<div class="container">
-  <a onclick="history.back()" type="reset" class="btn btn-sm btn-default">
-    Cancel
-  </a>
-  <br>
-  <br>
-  <div class="alert alert-dismissable alert-warning">
-    <button type="button" class="close" data-dismiss="alert">×</button>
-    <h4>Hey! You Now Setting for Group...</h4>
-    <p style="color:white">hint: HOLD shift to select more than two object</p>
-    <p style="color:white">hint: any change you made will save to the group</p>
+<div class="col-lg-offset-1 col-lg-10">
+  <h2>{{ group.id }} - {{ group.gname }}</h2>
```

Comment:
How about putting cancel and confirm button together? 
BTW other pages use save instead of confirm and there is a typo in your button name.